form/form_test.go

```diff
@@ -612,8 +673,15 @@ type response struct {
 }
 
 type testPrompter struct {
-	prompts   []prompt
-	responses []response
+	title         string
+	titleResponse response
+	prompts       []prompt
+	responses     []response
+}
+
+func (p *testPrompter) ShowTitle(title string) error {
+	p.title = title
```

Comment:
perhaps p.showTitleCalled = true ?
then you'll be able to tell if it's been called appropriately or not.